ci/docker/Dockerfile.build.jetson

```diff
@@ -90,6 +90,9 @@ RUN wget https://repo.download.nvidia.com/jetson/x86_64/pool/r32.4/c/cuda/cuda-c
 # while previously it used to store it at /usr/local/cuda/targets/aarch64-linux/lib/stubs
 RUN ln -s /usr/lib/aarch64-linux-gnu/libcublas.so /usr/local/cuda/targets/aarch64-linux/lib/stubs/libcublas.so
 
+# Need the stub for libcuda.so in a place where host linker can find it to link the unit tests
+RUN ln -s /usr/local/cuda/targets/aarch64-linux/lib/stubs/libcuda.so /usr/lib/aarch64-linux-gnu/libcuda.so
```

Comment:
Do you know why `/usr/local/cuda/targets/aarch64-linux/lib/stubs/libcuda.so` is not detected by cmake? It shouldn't be necessary to create this symlink (it isn't necessary in the non-cross-compile setting). Have you reported a bug in the cmake repository?

Reply:
It is detected by cmake, but only for CUDA compilation - the problem happens during the compilation of mxnet_unit_tests, which is not CUDA-aware I believe.

Reply:
`mxnet_unit_tests` is a cmake target that depends on the `libmxnet` target. The libraries detected when compiling `libmxnet` will also be propagated to `mxnet_unit_tests`. I think the problem is that you just added the `cuda` as dependency, instead of the correct `CUDA::cuda_driver` found by https://gitlab.kitware.com/cmake/cmake/-/blob/master/Modules/FindCUDAToolkit.cmake.

Reply:
I'm referring to this part of the cmakelists:

```
  list(APPEND mxnet_LINKER_LIBS CUDA::cudart CUDA::cublas CUDA::cufft CUDA::cusolver CUDA::curand)
  if(ENABLE_CUDA_RTC)
    if(CUDA_nvrtc_LIBRARY)
      list(APPEND mxnet_LINKER_LIBS CUDA::nvrtc cuda)
      add_definitions(-DMXNET_ENABLE_CUDA_RTC=1)
    else()
      message(FATAL_ERROR "ENABLE_CUDA_RTC=ON, but failed to find NVRTC. CMake will exit." )
    endif()
  endif()
  list(APPEND SOURCE ${CUDA})
  add_definitions(-DMXNET_USE_CUDA=1)
```

Reply:
Ok, let me try changing it.

Reply:
Seems to be working :-). Thanks for the suggestion!